packages/mask/src/plugins/Pets/SNSAdaptor/NormalNFT.tsx

```diff
@@ -94,6 +123,14 @@ export function NormalNFT(props: NormalNFTProps) {
             {infoShow ? (
                 <div className={classes.close} onClick={handleClose} style={{ backgroundImage: `url(${CloseIcon})` }} />
             ) : null}
+            <div className={classes.fullBox} onContextMenu={handleMenuShow} />
```

Comment:
Why a `div` node has got `onContextMenu` method?

Reply:
yes, can be removed